src/thirdparty/caman/caman.full.min.js

```diff
@@ -0,0 +1,244 @@
+/*
```

Comment:
It's fine if this happens in a follow-up, it doesn't need to block this from landing.